incubator/monochart/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A declarative helm chart for deploying common types of services on Kubernetes
 name: monochart
-version: 0.15.0
-appVersion: 0.15.0
+version: 0.16.0
```

This is a small enough change, fully backward compatible, that it can be a patch release.
```suggestion
version: 0.15.1
```